submissions/natashafir/html-css-popup/index.html

```diff
@@ -0,0 +1,80 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link href="./css/style.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand&display=swap" rel="stylesheet">
+</head>
+<body>
+<div class="nav">
+    <ul class="nav__social social">
+        <li class="nav-item"><a href="#" class="social__item mail" tabindex="0"> Mail</a></li>
+        <li class="nav-item"><a href="#" class="social__item img" tabindex="0">Images</a></li>
+        <li class="nav-item">
+            <div class="social__item menu">
+
+                <input type="checkbox" id="popup">
+                <label for="popup" class="popup-label"></label>
+
+                <div class="popup-menu">
+                    <a href="#" class="popup-item">
+                        <img src="./img/html.png" alt="">
```

Comment:
It's not a blocker, but don't forget about setting the alternative text for your images in the `alt` attribute

Reply:
Thank you.